lib/Parse/Lexer.cpp

```diff
@@ -1204,6 +1206,68 @@ static bool maybeConsumeNewlineEscape(const char *&CurPtr, ssize_t Offset) {
   }
 }
 
+/// diagnoseZeroWidthMatchAndAdvance - Error zerowidth characters in delimiters.
+/// A non visible character in the middle of a delimter can be used to extend
```

Comment:
Nits: “zero-width” instead of “zerowidth,” “invisible” instead of “non visible.”

Typo in the word “delimiter.”